src/services/get-env-api.js

```diff
@@ -1,8 +1,24 @@
 import store from "../store";
 import constants from "../utils/constants";
 
-export default function () {
-  return store.getters.env === "test"
-    ? constants.api.testnet
-    : constants.api.mainnet;
+const CURRENT_API_VERSION = import.meta.env.VITE_CURRENT_API_VERSION;
+
+export default function (version) {
```

Comment:
looks like the value of version is either be `false` or **version number**, I suggest to set a default value.

#### For example:
```
function (version = CURRENT_API_VERSION) {
}
```

Reply:
Sounds good

Reply:
[nit] Prefer exporting a named function. If there's an error thrown, the stack trace will include the function name and help with finding the source of the error.

```js
export default function getEnvApi(version) {
  ...
}
```